pkgs/applications/video/kodi-packages/jellyfin/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, addonDir, buildKodiAddon, fetchFromGitHub, python3, requests, dateutil, six, kodi-six, signals }:
+let
+  python = python3.withPackages (p: with p; [ pyyaml ]);
```

Comment:
I think you are potentially mixing python versions/environments here.

Reply:
I thought since it is just being used as a `nativeBuildInputs` dependency it shouldn't matter... but even if that is correct you make a good point: why bother pulling in multiple versions of `python`?

Thanks @SuperSandro2000!

Reply:
It is usually not that easy to debug so it should be avoided even if it makes no difference.

Reply:
Definitely good points :+1: